### ISSUE TITLE
Google Drive Access blocked

### ISSUE THREAD

**Issue by madbor** [PhilippC/keepass2android#1899]
I have my Keypass database on Google Drive. Recently, when I try to save or update an entry from Keypass2Android I get this error message: "Could not save source file: Built-in Google Drive access is currently blocked by Google. Please u..." (Rest of message is not showing - does anybody know what the rest of the message is?) 
My version is 1.09b-r1. Android is 12.

I can see that the kdbx file on GD was last modified on December 28th - so something has recently changed.

Is this an Keypass2Android issue that will/can be fixed, or is it a Google security thing that, from now on, will not allow apps like Keypass2Android to connect to GD?

Cheers

Peter

**Comment by madbor** [comment]
Right then, I found a workaround that will allow access to GD. It is sort of described in the app already, but not easy to find. Here are the steps:

1) From the Keypass2Android app select "Open another database" in the 3dot menu.
2) select "Open File"
3) select "System File Picker" from the tile with the Android robot logo.
4) Select the Google Drive icon associated with you Google account
5) Select your kdbx database file from GD.

After completing this, I tested it by modifying an entry, saved it and then the file edit date on GD updated accordingly.

I hope this works for you as well.
